inference-engine/src/vpu/graph_transformer/src/model/model.cpp

```diff
@@ -1908,14 +1955,10 @@ void ModelObj::removeStage(const Stage& stage) {
 }
 
 void ModelObj::cleanUp() {
-    bool needAllocatorPreprocess = false;
```

Comment:
This flag is set only in `cleanUp` in case 
```
if (data->usage() != DataUsage::Intermediate && data->usage() != DataUsage::Temp) { 
    needAllocatorPreprocess = true;
}
```
and then read in the only place
```
if (needAllocatorPreprocess) {	
        _allocator.setNeedToAllocNonIntermData();	
}
```
Code in `removeUnusedData()` do the same
```
if (data->usage() != DataUsage::Intermediate && data->usage() != DataUsage::Temp) {	        
        _allocator.setNeedToAllocNonIntermData();	      
}
```